{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "Intel(R) Graphics Drivers",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "before version 15.33.51.5146"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Uncaught exception in some Intel(R) Graphics Drivers before version 15.33.51.5146 may allow an authenticated user to potentially enable denial of service via local access."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "denial of service",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2021-02-17T13:52:06",
                "orgId": "6dda929c-bb53-4a77-a76d-48e79601a1ce",
                "shortName": "intel"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00438.html"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "6dda929c-bb53-4a77-a76d-48e79601a1ce",
        "assignerShortName": "intel",
        "cveId": "CVE-2020-24448",
        "datePublished": "2021-02-17T13:52:06",
        "dateReserved": "2020-08-19T00:00:00",
        "dateUpdated": "2021-02-17T13:52:06",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
